code/EmogrifiedSmtpMailer.php

```diff
@@ -163,7 +163,10 @@ public function sendHTML($to, $from, $subject, $htmlContent, $attachedFiles = fa
         }
         if ($level = $this->getSMTPDebug()) {
             $mail->SMTPDebug = $level;
-            $mail->Debugoutput = 'html';  // HTML friendly report
+            $mail->Debugoutput = function ($str, $level)
+            {
+                SS_Log::log(new Exception(print_r($str, true)), SS_Log::NOTICE);
```

Comment:
I don't think you have to use an exception to log anymore. Could we just send it the output of print_r?